app/assignment/services/assignmentActionCreators.js

```diff
@@ -206,6 +174,35 @@ assignmentApp.service('assignmentActionCreators', function (assignmentStateServi
 				$rootScope.$emit('toast', { message: "Could not assign instructor to course.", type: "ERROR" });
 			});
 		},
+		assignInstructorType: function (teachingAssignment) {
+			var scheduleId = assignmentStateService._state.userInterface.scheduleId;
+
+			assignmentService.addInstructorAssignment(teachingAssignment, scheduleId).then(function (newTeachingAssignment) {
+				$rootScope.$emit('toast', { message: "Assigned instructor type", type: "SUCCESS" });
+				assignmentStateService.reduce({
+					type: ADD_TEACHING_ASSIGNMENT,
+					payload: {
+						teachingAssignment: newTeachingAssignment
+					}
+				});
+			}, function (err) {
+				$rootScope.$emit('toast', { message: "Could not assign instructor to course.", type: "ERROR" });
```

Comment:
Success is 'assigned instructor type' but failure is 'could not assign instructor to course'? Aren't those slightly different?